package.json

```diff
@@ -8,7 +8,8 @@
     "lint": "eslint \"src/**/*.{js,jsx}\"",
     "stylelint": "stylelint src",
     "prettier": "prettier --check \"src\"",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "start": "vue-cli-service serve"
```

Comment:
🔧 Can we switch this to `vite`?
https://vitejs.dev/guide/static-deploy.html

Reply:
It appears that the vue-cli-service serve wasn't needed, and I deleted it.